### ISSUE TITLE
VS Code Azure Extension and Azure Gov Support

### ISSUE THREAD

**Issue by wayneme75** [microsoft/vscode#200088]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.8.4.2
- OS Version: Mac OS Darwin arm64 23.1.0

Version: 1.84.2 (Universal)
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:52:33.687Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: 14.1.2 (23B92)

Steps to Reproduce:

1. Install extension
2. Change settings to use Azure Government
3. Attempt sign-on from the Azure extension
4. Failed login from VS Code in Azure:
5. Failure reason: Consent between first party application '{applicationId}' and first party resource '{resourceId}' must be configured via preauthorization - applications owned and operated by Microsoft must get approval from the API owner before requesting tokens for that API.
6. Additional details: A developer in your tenant may be attempting to reuse an App ID owned by Microsoft. This error prevents them from impersonating a Microsoft application to call other APIs. They must move to another app ID they register in portal.azure.com.
7. App ID which is being referenced: aebc6443-996d-45c2-90f0-388ff96faa56 
8. Resource: Microsoft Graph
9. Resource ID: 00000003-0000-0000-c000-000000000000

This used to work without issue, stopped working in the last week.

**Comment by TylerLeonhardt** [comment]
Looks like https://github.com/microsoft/vscode/issues/198562 ... @alexweininger @bwateratmsft do you know more about this?

**Comment by alexweininger** [comment]
Yeah see https://github.com/microsoft/vscode-azureresourcegroups/issues/785 for details. It's fixed in main and we are waiting on testing to release it. Please roll back to Azure Resources 0.7.5 in the meantime.

**Comment by TylerLeonhardt** [comment]
Closing in favor of that issue
